docs/internal/release.md

```diff
@@ -101,119 +100,86 @@
-    - On OS X do: Finder->Go->Connect to Server...  Then connect to `smb://www/web`.
-
-17. Set the `webdir` and `ftpdir` Gradle properties
-    - Open `~/.gradle/gradle.properties`
-    - Set `webdir` to `${webMountDir}/content/software/netcdf-java/v${releaseMajor}`
-    - Set `ftpdir` to `${webMountDir}/ftp/pub/netcdf-java/v${releaseMajor}`
-    - The value of `webMountDir` will likely differ, but mine is `/Volumes/web`.
-    - So for example, on OS X, my (Christian's) Gradle properties for `webdir` and `ftpdir` are:
-      ```properties
-      webdir=/Volumes/web/content/software/netcdf-java/v4.6
-      ftpdir=/Volumes/web/ftp/pub/netcdf-java/v4.6
-      ```
-
-18. Release Web Start to `www:/content/software/netcdf-java/v${releaseMajor}/webstart`
+16. Release Web Start to `downloads.unidata.ucar.edu/netcdf-java/4.6/webstart`
     - Make sure that you have the correct gradle.properties (see Christian for info). In particular, you'll need the
-      `keystore`, `keystoreAlias`, `keystorePassword`, `webdir`, and `ftpdir` properties defined.
-    - Rename old directories
-      * `cd /content/software/netcdf-java/v${releaseMajor}/`
-      * `mv webstart webstartOld`
-    - Perform release
-      * `./gradlew :ui:clean :ui:releaseWebstart`
-    - Test the new Web Start. If there were no errors, delete the old stuff.
-      * `rm -r webstartOld`
-
-19. Release Javadoc to `www:/content/software/netcdf-java/v${releaseMajor}/javadoc` and `javadocAll`
-    - Rename old directories
-      * `cd /content/software/netcdf-java/v${releaseMajor}/`
-      * `mv javadoc javadocOld`
-      * `mv javadocAll javadocAllOld`
-    - Perform release
-      * `./gradlew :cdm:clean :cdm:releaseDocs`
-      * `./gradlew :ui:clean :ui:releaseDocs`
-    - If there were no errors and the new Javadoc looks good, delete the old stuff.
-      * `rm -r javadocOld`
-      * `rm -r javadocAllOld`
-
-20. Change permissions of the files you just copied.
-    ```bash
-    cd /content/software/netcdf-java/v${releaseMajor}/
-    find webstart -type d -exec chmod 775 {} \;
-    find webstart -type f -exec chmod 664 {} \;
-    find javadoc -type d -exec chmod 775 {} \;
-    find javadoc -type f -exec chmod 664 {} \;
-    find javadocAll -type d -exec chmod 775 {} \;
-    find javadocAll -type f -exec chmod 664 {} \;
-    ```
-
-21. Update Unidata download page(s)
+      `keystore`, `keystoreAlias`, and `keystorePassword` properties defined.
+    - Remove old webstart files
+      * `./gradlew -PdryRun=false :ui:deleteWebstart`
+    - Publish new webstart files
+      * `./gradlew clean :ui:publishWebstart`
+
+17. Release documentation and javadocs to nexus
+    - remove old 4.6 docs
+      * `./gradlew -PdryRun=false :docs:website:deleteAllDocs`
+    - upload new 4.6 docs
+      * `./gradlew :docs:website:publishAllDocs`
+
+18. Update Unidata download page(s)
     - check https://www.unidata.ucar.edu/downloads/tds/
```

Comment:
Let's remove the reference to toc.xml as the downloads isn't used anymore for the THREDDS project.

Reply:
👍